app/src/navigation/screens/ClaimScreen.tsx

```diff
@@ -152,10 +151,20 @@ const ClaimScreen: React.FC = () => {
   };
   const openVerifyOtpScreen = async () => {
     setLoading(true);
-    const { countryCode, number } = formatMobileNumberState();
+
+    const newMobileState = formatMobileNumberState();
+    if (newMobileState.countryCode !== "+61") {
```

Comment:
Sould we trim to be sure?

```suggestion
    if (newMobileState.countryCode.trim() !== "+61") {
```

Reply:
Added the trim in :)